tests/unit/configuration/test_inheritance_break_validation.py

```diff
@@ -20,8 +20,9 @@ def test__validate_break_inheritance_flag__invalid_flag_set_at_common_level(self
                 value: bar
         """
 
+        configuration = Configuration(config_string=config_yaml)
         with pytest.raises(SystemExit) as exception:
-            Configuration(config_string=config_yaml).get_common_config()
+            configuration.get_common_config()
```

Comment:
`get_common_config` doesn't do any validation, so there wouldn't be any error.

Reply:
It does since my commit https://github.com/gitlabform/gitlabform/pull/385/commits/c8b36177b7b2175d847557fcca82cd22a78c5dd2

Reply:
Oh okay, let me further investigate today.

Reply:
Okay, I see what the problem is. It goes back to this [comment here](https://github.com/gitlabform/gitlabform/pull/385#discussion_r904517085) regarding the inheritance-break validation not making sense. 

Going with what you have in the MR, I think we should remove the [invalid group test](https://github.com/ep-linden/gitlabform/pull/1/commits/1b0722b1c97dce05c21bde8d371bcc6bbf78b065#diff-d82e9553838cd98ea73844b7a2d2c20fb2c675c2db9113448def3a8ba2648938R29) because it would be misleading now that I think about it.  

For the subgroup, instead of `get_effective_config_for_group`, it should be `get_effective_config_for_project` and change the subgroup config accordingly in the test. The funny thing is `get_effective_config_for_project` calls `get_effective_subgroup_config`, which resides in the `groups.py` file. That is the reason why there's no error coming from `get_effective_config_for_group`.